mgmt_upgrade_test.py

```diff
@@ -42,6 +43,11 @@ def test_upgrade(self):  # pylint: disable=too-many-locals,too-many-statements
         target_upgrade_agent_version = self.params.get('target_scylla_mgmt_agent_repo')
         manager_node = self.monitors.nodes[0]
         manager_tool = get_scylla_manager_tool(manager_node=manager_node)
+
+        with manager_tool.remote_scylla_manager_yaml() as scylla_manager_ymal:
+            node_ip = scylla_manager_ymal["http"].split(":", maxsplit=1)[0]
+            scylla_manager_ymal["http"] = f"{node_ip}:{self.params['manager_prometheus_port']}"
```

Comment:
this is not the correct port to change, `manager_prometheus_port`, `manager_port` should be a different one

Reply:
Done
```
 scylla_manager_ymal["http"] = f"{node_ip}:{self.params['mgmt_port']}"
            scylla_manager_ymal["prometheus"] = f"{node_ip}:{int(self.params['mgmt_port']) + 1}"
```

Reply:
no, if there's a SCT configuration for it, please use that `manager_prometheus_port`

Reply:
I did it